initializr-service/src/main/resources/application.yml

```diff
@@ -79,6 +79,14 @@ initializr:
           - versionRange: "2.0.0.BUILD-SNAPSHOT"
             version: Finchley.BUILD-SNAPSHOT
             repositories: spring-snapshots,spring-milestones
+      spring-cloud-gcp:
+        groupId: org.springframework.cloud
+        artifactId: spring-cloud-gcp-dependencies
+        versionProperty: spring-cloud-gcp.version
+        mappings:
+          - versionRange: "2.0.0.RC2"
```

Comment:
That's the version range of Spring Boot and we have no requirements to support multiple Spring Cloud GCP versions at the moment. I'd just add the version as a simple property for now.